README.md

```diff
@@ -286,6 +286,20 @@ Pretty.prettyPrint(value, option: Pretty.Option(prefix: nil, indentSize: 2))
 // )
 ```
 
+### colorized
+
+Output strings can be colored.
+
+```swift
+Pretty.sharedOption = Pretty.Option(colored: true)
+let value = (bool: true, array: ["Hello", "World"])
+Pretty.prettyPrint(value)
+```
+
+This output looks like as bellow:
+
+<img src=https://raw.githubusercontent.com/YusukeHosonuma/SwiftPrettyPrint/main/Image/colored-option.png>
```

Comment:
Please use relative-path to reference image.

Reply:
Fine. Fixed.

8230ed69264b7c5b2daf97f2c089773be101ffb7

On the other hand, some of the existing descriptions in REAEME.md are written in absolute paths as in the pointed out sections.

I've created a separate PR #174 to eliminate that confusion, so please review that as well.
@YusukeHosonuma